apps/web/src/app/(private)/evaluations/[evaluationUuid]/editor/_components/EvaluationEditor/Editor/Playground/index.tsx

```diff
@@ -80,7 +107,13 @@ export default function Playground({
         Array.from(metadata.parameters).map((param) => {
           if (param in inputs) return [param, inputs[param]!]
 
-          return [param, { value: '', type: 'input' }]
+          return [
+            param,
+            {
+              value: '',
+              type: param in EVALUATION_PARAMETERS ? 'textarea' : 'input',
+            },
+          ]
         }),
       ),
     )
```

Comment:
There shouldn't be ANY parameters that are not in `EVALUATION_PARAMETERS` when running an evaluation.